modules/obfuscating-dvo-data.adoc

```diff
@@ -7,7 +7,12 @@
 [id="obfuscating-deployment-validation-operator-data_{context}"]
 = Obfuscating Deployment Validation Operator data
 
-Cluster administrators can configure the Insights Operator to obfuscate data from the Deployment Validation Operator (DVO), if the Operator is installed. When the `workload_names` value is added to the *insights-config* `ConfigMap` object, workload names—rather than UIDs—are displayed in Insights for Openshift, making them more recognizable for cluster administrators. 
+By default, when you install DVO, the name and UID of a resource are displayed by the Insights Operator for {product-title}. 
```

Comment:
The data is not really displayed by the operator, but it's present in the Insights archive sent to RH.

Reply:
Thanks @tremes 👍 

How about ...

_By default, when you install the Deployment Validation Operator (DVO), the name and UID of a resource are included in the data that is captured and processed by the Insights Operator for OpenShift. If you are a cluster administrator, you can configure the Insights Operator to obfuscate data from the Deployment Validation Operator (DVO). For example, you can obfuscate workload names in the archive file that is then sent to Red Hat._

Reply:
Looks good to me. Thank you!